profiles/kentik_snmp/_general/ups-mib.yml

```diff
@@ -0,0 +1,163 @@
+# Profile is intended to be used in an extends block
+# https://www.circitor.fr/Mibs/Html/U/UPS-MIB.php
+---
+metrics:
+  - MIB: UPS-MIB
+    symbols:
+      # The indication of the capacity remaining in the UPS system's batteries. A value of batteryNormal indicates that the remaining run-time is greater than upsConfigLowBattTime. A value of batteryLow indicates that the remaining battery run-time is less than or equal to upsConfigLowBattTime. A value of batteryDepleted indicates that the UPS will be unable to sustain the present load when and if the utility power is lost (including the possibility that the utility power is currently absent and the UPS is unable to sustain the output).
+      - OID: 1.3.6.1.2.1.33.1.2.1.0
+        name: upsBatteryStatus
+        enum:
+          unknown: 1
+          batteryNormal: 2
+          batteryLow: 3
+          batteryDepleted: 4
+      # If the unit is on battery power, the elapsed time since the UPS last switched to battery power, or the time since the network management subsystem was last restarted, whichever is less. Zero shall be returned if the unit is not on battery power.
+      - OID: 1.3.6.1.2.1.33.1.2.2.0
+        name: upsSecondsOnBattery
+      # An estimate of the time to battery charge depletion under the present load conditions if the utility power is off and remains off, or if it were to be lost and remain off.
+      - OID: 1.3.6.1.2.1.33.1.2.3.0
+        name: upsEstimatedMinutesRemaining
+      # An estimate of the battery charge remaining expressed as a percent of full charge.
+      - OID: 1.3.6.1.2.1.33.1.2.4.0
+        name: upsEstimatedChargeRemaining
+      # The magnitude of the present battery voltage.
+      - OID: 1.3.6.1.2.1.33.1.2.5.0
+        name: upsBatteryVoltage
+      # The present battery current.
+      - OID: 1.3.6.1.2.1.33.1.2.6.0
+        name: upsBatteryCurrent
+      # The ambient temperature at or near the UPS Battery casing.**
+      - OID: 1.3.6.1.2.1.33.1.2.7.0
+        name: upsBatteryTemperature
+      # A count of the number of times the input entered an out-of-tolerance condition as defined by the manufacturer. This count is incremented by one each time the input transitions from zero out-of-tolerance lines to one or more input lines out-of-tolerance.
+      - OID: 1.3.6.1.2.1.33.1.3.1.0
+        name: upsInputLineBads
+      # The number of input lines utilized in this device. This variable indicates the number of rows in the input table.
+      - OID: 1.3.6.1.2.1.33.1.3.2.0
+        name: upsInputNumLines
```

Comment:
You know I no longer have confidence in the uniquecount() function :) 

kidding aside, it's potentially noise but I didn't see harm in including it if the mib provides it instead of having to calculate the value in NRQL